diff --git a/lib/runtime/desktop_xray_runtime.dart b/lib/runtime/desktop_xray_runtime.dart
index 5730255..758c8f6 100644
--- a/lib/runtime/desktop_xray_runtime.dart
+++ b/lib/runtime/desktop_xray_runtime.dart
@@ -3,6 +3,7 @@ import 'dart:convert';
 import 'dart:io';
 import 'dart:typed_data';
 
+import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
 import 'package:path_provider/path_provider.dart';
 
